base/services/wzcsvc/wzcsvc.c

```diff
@@ -0,0 +1,162 @@
+/*
+ * PROJECT:          ReactOS System Services
+ * LICENSE:          GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * FILE:             base/services/wzcsvc/wzcsvc.c
+ * PURPOSE:          ReactOS Wireless Zero Configuration Service
+ * COPYRIGHT:        Copyright 2020 Oleg Dubinskiy (oleg.dubinskij2013@yandex.ua)
+ */
+
+/* INCLUDES *****************************************************************/
+
+#define WIN32_NO_STATUS
+#define _INC_WINDOWS
+#define COM_NO_WINDOWS_H
+#include <stdarg.h>
+#include <windef.h>
+#include <winbase.h>
+#include <winreg.h>
+#include <winsvc.h>
+
+#define NDEBUG
+#include <debug.h>
+
+#include <svc.h>
```

Comment:
How does this build for you, where does it come from?

Reply:
This header contains all global svchost declarations which is my work from my #2931 PR.
This is required e. g. by some services, which implement SvchostPushServiceGlobals function.